sdks/python/setup.py

```diff
@@ -30,8 +30,7 @@
 
 def get_version():
   global_names = {}
-  execfile(os.path.normpath('./apache_beam/version.py'),
-           global_names)
+  exec(open(os.path.normpath('./apache_beam/version.py')).read(), global_names)
```

Comment:
Right, the trick with `exec` is for avoiding importing the `beam` module upon its installation.